src/main/plugins/org.dita.pdf2/xsl/fo/commons.xsl

```diff
@@ -459,6 +459,9 @@ See the accompanying license.txt file for applicable licenses.
 
     <xsl:template name="processTopicNotices">
         <fo:page-sequence master-reference="body-sequence" xsl:use-attribute-sets="page-sequence.notice">
+            <xsl:if test="key('map-id', ancestor-or-self::*[contains(@class, ' topic/topic ')][1]/@id)/ancestor::*[contains(@class,' bookmap/backmatter ')]">
```

Comment:
My preference is that we don't introduce a new mode in the patch release. Instead, use the approach IMO should be used to replace attribute reflection. (Attribute reflection is a result of XSLT 1 limitations, with XSLT 2 we no longer need it):

```
<xsl:template name="processTopicNotices">
  <xsl:variable name="atts" as="element()">
    <xsl:choose>
      <xsl:when test="back matter">
        <dummy xsl:use-attribute-sets="page-sequence.backmatter.notice"/> 
      </xsl:when>
      <xsl:otherwise>
        <dummy xsl:use-attribute-sets="page-sequence.notice"/> 
      </xsl:otherwise>
    </xsl:choose>
  </xsl:variable>
  <fo:page-sequence master-reference="body-sequence">
    <xsl:copy-of select="$atts/@*"/>
    ...
```

It's a bit verbose, but unfortunately that's the only way we can decide between attribute sets.